cpp/include/Ice/LoggerUtil.h

```diff
@@ -129,8 +130,14 @@ namespace Ice
     class ICE_API Trace : public LoggerOutputBase
     {
     public:
-        Trace(LoggerPtr, std::string);
+        /// Constructs a Trace object with the given logger and category.
+        /// @param logger The logger to log to.
+        /// @param category The trace category.
+        Trace(LoggerPtr logger, std::string category);
+
         ~Trace();
+
+        /// Calls Logger#trace with the collected output.
```

Comment:
Assuming `trace` is a function here, we prefer `::`:
```suggestion
        /// Calls Logger::trace with the collected output.
```
Doxygen tolerates '#' anywhere where `::` could appear for JavaDoc compatibility, but we shouldn't abuse this.

We only use `#` for fields/enumerators, or for referencing the 'current scope', ie '#Thing' will point to something named `Thing` in the same scope as the doc-comment. Anywhere else, we should use `::`.